spec/requests/graphql/mutations/digital_object/update_descriptive_metadata_mutation_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Updating Item Descriptive Metadata', type: :request, solr: true do
+  let(:project) { FactoryBot.create(:project) }
+  let(:authorized_item) { FactoryBot.create(:item, primary_project: project) }
+
+  let(:field_definitions) do
+    {
+      dynamic_field_categories: [{
+        display_label: "Descriptive Metadata",
+        metadata_form: 'descriptive',
+        dynamic_field_groups: [
+          {
+            string_key: 'title',
+            display_label: 'Title',
+            dynamic_fields: [
+              { display_label: 'Non-Sort Portion', sort_order: 1, string_key: 'non_sort_portion', field_type: DynamicField::Type::STRING },
+              { display_label: 'Sort Portion', sort_order: 2, string_key: 'sort_portion', field_type: DynamicField::Type::STRING }
+            ]
+          }
+        ]
+      }]
+    }
+  end
+
+  before do
+    Hyacinth::DynamicFieldsLoader.load_fields!(field_definitions)
+    search_params = { digital_object_type: 'item' }
+    EnabledDynamicField.where(search_params).delete_all
```

Comment:
Same question here about why `delete_all` is necessary.

Reply:
```suggestion
```